demo/config/tools/cmd_tlm_server/cmd_tlm_server.txt

```diff
@@ -1,6 +1,6 @@
 TITLE 'COSMOS Command and Telemetry Server - Demo Configuration'
 
-PACKET_LOG_WRITER DEFAULT meta_packet_log_writer.rb META DATA config/data/meta_init.txt
+PACKET_LOG_WRITER DEFAULT packet_log_writer.rb nil true nil 2000000000 nil false config/data/meta_init.txt
```

Comment:
I'll comment in the config file.   Thinking through this made me realize the meta init file should probably be handled in system.txt, not the log writer